includes/payments/class-payments-query.php

```diff
@@ -512,43 +512,19 @@ public function give_forms() {
 			return;
 		}
 
-		global $give_logs;
-
-		$args = array(
```

Comment:
@DevinWalker I did not find the use of this code inside core but it is working fine with give-cli.